pkg/cmd/github-pull-request-make/main.go

```
@@ -126,6 +126,31 @@ func pkgsFromDiff(r io.Reader) (map[string]pkg, error) {
 	}
 }
 
+func findPullRequest(
+	ctx context.Context, client *github.Client, org, repo, sha string,
+) *github.PullRequest {
+	opts := &github.PullRequestListOptions{
+		ListOptions: github.ListOptions{PerPage: 100},
+	}
+	for {
+		pulls, resp, err := client.PullRequests.List(ctx, org, repo, opts)
+		if err != nil {
+			log.Fatal(err)
```

seems better to return this error